clayui.com/src/components/LayoutNavHome/index.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * SPDX-FileCopyrightText: © 2018 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import LayoutNavHome from './LayoutNavHome';
+
+export default LayoutNavHome;
```

Comment:
If you take a look at other similar components in `clayui.com/src/components` like `IconSearch` and `CodeToggle` you can see the exporting pattern that doesn't require 2 files: `LayoutNavHome.js` and `index.js` files but just the index file.